resultstore-search-api/credentials.py

```diff
@@ -37,9 +36,10 @@ def __init__(self):
         self.project_id = os.environ.get('PROJECT_ID')
         self.client_id = os.environ.get('CLIENT_ID')
         self.destination_sever = os.environ.get('DESTINATION_SERVER')
```

Comment:
Didn't catch this earlier. It's destination_server. 

I would recommend documenting here what each of these fields is meant for.

Also, rename DESTINATION_SERVER to RESULTSTORE_API_ENDPOINT.